packages/frontend/web/components/SubMetaLinksList.tsx

```diff
@@ -77,9 +72,7 @@ export const SubMetaLinksList: React.FC<{
 }> = ({ links, isSectionLinkList, pillar }) => (
     <ul
         className={cx(subMetaLinksList, [
-            isSectionLinkList
-                ? subMetaSectionLinksList
-                : subMetaKeywordLinksList,
+            !isSectionLinkList && subMetaKeywordLinksList,
```

Comment:
The biggest refactor of the lot... Basically line height was only used in the one class, so having to move from teneray to single falsey check